src/bindings/python/flux/core/handle.py

```diff
@@ -11,7 +11,6 @@
 import signal
 import threading
 from contextlib import contextmanager
-import six
```

Comment:
Does this commit remove every reference to `six` in the codebase?  If so, can you remove the check for `six` in the configure.ac: https://github.com/flux-framework/flux-core/blob/master/configure.ac#L253

And also remove `six` from the dependency table in the README.md: https://github.com/flux-framework/flux-core/blame/master/README.md#L59

Reply:
Thanks! I would never have thought of that. I got rid of all the other references to six and then fixed up the README.md and configure.ac.